assets/src/edit-story/components/panels/pageBackground.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState, useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app';
+import { useTheme } from '../../theme';
+import { InputGroup } from '../form';
+import { SimplePanel } from './panel';
+
+function PageBackgroundPanel() {
+  const {
+    state: { currentPage },
+    actions: { updateCurrentPageProperties },
+  } = useStory();
+  const theme = useTheme();
+  const defaultColor = theme.colors.fg.v1;
+  const [color, setColor] = useState(
+    currentPage?.backgroundColor || defaultColor
+  );
+  useEffect(() => {
+    setColor(currentPage?.backgroundColor || defaultColor);
+  }, [currentPage, defaultColor]);
```

Comment:
Should the deps instead be `[currentPage?.backgroundColor, ...]` as well? Otherwise the change of data outside will not update the panel.

Reply:
`currentPage` should be considered an immutable and will only remain identical by reference, if all values are identical.

But it does make sense to extract the value and depend on it directly to not re-render this just because something else changes on the current page.

Reply:
Fixed in b0c9c36b.

Reply:
good point.